cmake/tests.cmake

```diff
@@ -4,6 +4,7 @@ endif()
 
 option(protobuf_ABSOLUTE_TEST_PLUGIN_PATH
   "Using absolute test_plugin path in tests" ON)
+mark_as_advanced(protobuf_ABSOLUTE_TEST_PLUGIN_PATH)
```

Comment:
I generally reserve mark_as_advanced for options that are not reccomended, but are user-facing, or for the myriad of <package-name>_* variables that are created by Find modules. I'm fine either way though.